benchmark/timers/timers-timeout-unpooled.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+const common = require('../common.js');
+
+// The following benchmark sets up n * 1e6 unpooled timeouts,
+// then measures their execution on the next uv tick
+
+const bench = common.createBenchmark(main, {
+  millions: [1],
```

Comment:
I personally do not like the "millions" and would actually like to get replace those with e.g. `n` over time but that might only be my view on it. It is at least for me easier to read how many operations got executed that way instead of having to see that the output is actually also in millions.

Reply:
Thanks for the feedback. Changed over to `n = 1e6`.